dbms/src/Storages/MergeTree/ReplicatedMergeTreePartCheckThread.cpp

```diff
@@ -233,7 +233,7 @@ void ReplicatedMergeTreePartCheckThread::checkPart(const String & part_name)
                 zk_part_header.getChecksums().checkEqual(local_part_header.getChecksums(), true);
 
                 checkDataPart(
-                    storage.data.getFullPath() + part_name,
+                    storage.data.getFullPaths()[0] + part_name, ///@TODO_IGR ASK what should we do there? Should we check all paths in checkDataPart?
```

Comment:
No, we should check part `part_path`.

Reply:
How can we get part_path by part_name?
Or we shold chg queu by containig path in queue